speech/google/cloud/speech_v1/gapic/transports/speech_grpc_transport.py

```diff
@@ -93,7 +97,8 @@ def create_channel(cls, address="speech.googleapis.com:443", credentials=None):
             address,
             credentials=credentials,
             scopes=cls._OAUTH_SCOPES,
-            options={"grpc.max_receive_message_length": 256 * 1024 * 1024}.items(),
+            options={"grpc.max_receive_message_length": 256 * 1024 * 1024}.items()
+            ** kwargs,
```

Comment:
The generator is screwing up here:  the trailing comma is needed *before* `**kwargs`, not after.